src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -3342,6 +3770,7 @@ mod tests {
             overhead_panel.blue_epump_override_push_button.push_off();
 
             let mut blue_epump_controller = A320BlueElectricPumpController::new();
+            blue_epump_controller.is_powered = true;
```

Comment:
**issue**: Breaking encapsulation.
**suggestion**: If you want a pump to be powered, provide power! 😉